src/commands/synthetics/run-tests-command.ts

```diff
@@ -221,6 +243,22 @@ export class RunTestsCommand extends Command {
         subdomain: this.subdomain,
         testSearchQuery: this.testSearchQuery,
         tunnel: this.tunnel,
+        ...(this.allowedIPv4Ranges || this.allowedIPv6Ranges
+          ? {
+              allowedIPRanges: {
+                [IpFamily.v4]: this.allowedIPv4Ranges,
+                [IpFamily.v6]: this.allowedIPv6Ranges,
+              },
+            }
+          : {}),
+        ...(this.blockedIPv4Ranges && this.blockedIPv6Ranges
+          ? {
+              blockedIPRanges: {
+                [IpFamily.v4]: this.blockedIPv4Ranges,
+                [IpFamily.v6]: this.blockedIPv6Ranges,
+              },
+            }
+          : {}),
```

Comment:
Ah good catch, just a mistake. Would love to know any other way to do this so it feels less fragile

Reply:
I guess we could provide default values to the ranges provided in input, and then write everything without any conditions:
```typescript
{
  allowedIPRanges: {
    [IpFamily.v4]: this. allowedIPv4Ranges,
    [IpFamily.v6]: this. allowedIPv6Ranges,
  },
  blockedIPRanges: {
    [IpFamily.v4]: this. blockedIPv4Ranges,
    [IpFamily.v6]: this. blockedIPv6Ranges,
  },
}
```
It would allow to remove the other `|| []` when trying to access the ranges later on.